pkg/scanners/scannerv4/convert.go

```diff
@@ -109,6 +111,43 @@ func vulnerabilities(vulnerabilities map[string]*v4.VulnerabilityReport_Vulnerab
 	return vulns
 }
 
+func setCvss(vuln *storage.EmbeddedVulnerability, cvss *v4.VulnerabilityReport_Vulnerability_CVSS) error {
+	if cvss == nil {
+		return nil
+	}
+	errList := errorhelpers.NewErrorList("failed to parse vector")
+	if v2 := cvss.GetV2(); v2 != nil {
+		if c, err := cvssv2.ParseCVSSV2(v2.GetVector()); err == nil {
+			c.ExploitabilityScore = 0.0 // TODO cvssv2.ExploitabilityScore(v2.GetVector())
+			c.ImpactScore = 0.0         // TODO cvssv2.ImpactScore(v2.GetVector())
+			c.Score = v2.GetBaseScore()
+			c.Severity = cvssv2.Severity(v2.GetBaseScore())
```

Comment:
This sets the severity based on the CVSS scores. What do we do in other places? I think we try to set this to the given severity, if given

Reply:
> What do we do in other places?

What other places?

>  I think we try to set this to the given severity, if given

Maybe you're confusing the vulnerability severity from the CVSS severity?